apps/voteapp.yml

```diff
@@ -1,18 +1,37 @@
 # redis
---- 
+---
 apiVersion: v1
 kind: Service
-metadata: 
-  labels: 
+metadata:
+  name: voteapp-redis-service
+  labels:
+    app: voteapp-redis
+  annotations:
+    service.beta.kubernetes.io/azure-load-balancer-internal: "true"
+    service.beta.kubernetes.io/azure-pls-create: "true"
```

Comment:
With these annotations, we can no longer install VoteApp anywhere but in Azure? If that's the case, that's fine, but we probably want to cleanup the other assets & docs.

Reply:
I don't think it would stop the deployment from working (minus the missing worker pod).  More we would need different annotations to support the other clouds.